doc/6/getting-started/vuejs/without-vuex/index.md

```diff
@@ -0,0 +1,107 @@
+
+## Instancing Kuzzle SDK
+
+We have to connect the server so that our client can interact with it.
+
+To do this, we have to create _src/services/kuzzle.js_ file to put our kuzzle instance, a bit like a singleton:
+
+<<< ./src/services/kuzzle.js
+
+We need to import our Kuzzle SDK instance, so just add the following line in your `<script>` tag in the _App.vue_ file: 
+
+<<< ./src/App.vue:3[js]
+
+Then we will [etablish the connection](/sdk/js/6/core-classes/kuzzle/connect/) to kuzzle and create, if they don't [exists](sdk/js/6/controllers/index/exists/), create the [index](sdk/js/6/controllers/index/create/) and [collection](sdk/js/6/controllers/collection/create/) of our tchat. 
+Add the following `valid()` method in the export of the `<script>` tag of your _App.vue_ file:
+
+<<< ./src/App.vue:2[js]
+
+## Get the username
+First of all, we need to know the user pseudo. Let's start by adding the following html code in the `<template>` tag to allow the client type his pseudo:
+
+<<< ./src/App.vue:1[html]
+
+As you can see we'll need two properties: `username` and `validate`.
+
+
+## Display the messages
+
+We'll need some properties to manage our messages. Add the following `data` to your _App.vue_
+
+<<< ./src/App.vue:4[js]
+
+Then, create the following functions to fetch and display the messages: 
+
+<<< ./src/App.vue:5[js]
+
+<<< ./src/App.vue:6[js]
+
+The function `fetch_message()` will [search](/sdk/js/6/controllers/document/search/) for hundred newest existing messages and store them in our array before subscribe to notification about `messages` collection. We called it in the `valid()` function we created before.
+
+<<< ./src/App.vue:7[js]
+
+Now, just add the following html code to display the messages:
+
+<<< ./src/App.vue:8[html]
+
+To finish this part, just add the following css classes:
+
+<<< ./src/App.vue:9[css]
+
+We can now display the messages stored in Kuzzle. Cool but we need to create some right ?
+
+## Send messages
+
+We must make a simple method that will [create](/sdk/js/6/controllers/document/create/) a new message document in Kuzzle.
+
+<<< ./src/App.vue:10[js]
+
+As you can see we doesn't push the new message in our array on message creation.
+Actually, We'll subscribe to the collection that store our messages.
+So let's create our `subscribe_messages()` action. It will call the realtime controller of Kuzzle to allow us to [receive notifications](/sdk/js/6/controllers/realtime/subscribe/) on message creation:
+
+<<< ./src/App.vue:11[js]
+
+To finish, just add an input binded with the `message` property and a button wich call our `sendMessage()` function:
+
+<<< ./src/App.vue:12[html]
+
+And the following css classe: 
+
+<<< ./src/App.vue:13[css]
+
+You can now add new messages to Kuzzle, and receive the notification of the creation to update your store and display the new messages, enjoy :)
```

Comment:
Maybe you can add a section [Where do we go from here](http://docs.kuzzle.io/sdk/js/6/getting-started/node-js/#where-do-we-go-from-here)
and add a link to the realtime engine guide for example (http://docs.kuzzle.io/core/1/guides/essentials/real-time/